lib/modules/platform/azure/index.spec.ts

```diff
@@ -1251,6 +1251,28 @@ describe('modules/platform/azure/index', () => {
       expect(gitApiMock.createThread.mock.calls).toMatchSnapshot();
       expect(gitApiMock.updateComment.mock.calls).toMatchSnapshot();
     });
+
+    it('passes comment through massageMarkdown', async () => {
+      await initRepo({ repository: 'some/repo' });
+      const gitApiMock = {
+        createThread: jest.fn(() => [{ id: 123 }]),
+        getThreads: jest.fn().mockReturnValue([
+          {
+            comments: [{ content: 'end-user comment', id: 1 }],
+            id: 2,
+          },
+        ]),
+        updateComment: jest.fn(() => ({ id: 123 })),
+      };
+      azureApi.gitApi.mockImplementation(() => gitApiMock as any);
+      await azure.ensureComment({
+        number: 42,
+        topic: 'some-subject',
+        content:
+          'You can manually request rebase by checking the rebase/retry box above.',
+      });
+      expect(gitApiMock.createThread.mock.calls).toMatchSnapshot();
```

Comment:
Avoid `toMatchSnapshot()` and instead test for the important part within the test itself

Reply:
All right, a little explainer for myself is that I saw it being used quite often in this file. Anyway, I agree that mocking `massageMarkdown` and checking if it is called would be the best thing to do... however, it'd not easily possible because it is located in the same file, which is dynamically imported (`await import('.')`), so I can't mock it.

I think the best I can do is literally check if the string that is in the input is changed in a desired form. Hopefully you will agree, and if not then would it be enough to test `massageMarkdown` only?

Reply:
Actually you changed it exactly as I as meaning, so all is good - thanks